lib/node_modules/@stdlib/math/base/special/acscd/lib/main.js

```diff
@@ -0,0 +1,70 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var rad2deg = require('@stdlib/math/base/special/rad2deg');
+var acsc = require('@stdlib/math/base/special/acsc');
+
+
+// MAIN //
+
+/**
+* Computes the arccosecant (in degree) of a double-precision floating-point number.
+*
+* @param {number} x - input value
+* @returns {number} arccosecant (in degree)
+*
+* @example
+* var v = acscd( Infinity );
+* // returns 0.0
+*
+* @example
+* var v = acscd( 2 * Math.sqrt(3) / 3 );
+* // returns 60.00000000000001
+*
+* @example
+* var v = acscd( Math.sqrt(2) );
+* // returns 44.99999999999999
+*
+* @example
+* var v = acscd( 2 );
+* // returns 30.000000000000004
+*
+* @example
+* var v = acscd( 1 );
+* // returns 90.0
+*
+* @example
+* var v = acscd( NaN );
+* // returns NaN
+*/
+function acscd( x ) {
+	var rad;
+
+	rad = acsc(x);
+
+	return rad2deg(rad);
```

Comment:
When all variable declarations can be immediately assigned values, can collapse:

```suggestion
	var rad = acsc( x );
	return rad2deg( rad );
```

(notice also the added spaces around the function parentheses)